app/helpers/report_helper.rb

```diff
@@ -0,0 +1,7 @@
+module ReportHelper
+  def report_download_link(report)
+    return report.export_url if report.export_url
+
+    report_path(report, format: :csv)
```

Comment:
Is it simpler to say `report.export_url || report_path(report, format: :csv)`?

Reply:
I feel an early return is a bit more idiomatic Ruby and easily parseable rather than the `||` but don't have too strong an opinion on this!